Patterns/Spanish/Spanish-Numbers.yaml

```diff
@@ -190,6 +193,9 @@ OneNumberRangeMoreSeparateRegex: !nestedRegex
 OneNumberRangeLessRegex1: !nestedRegex
   def: ({LessOrEqual}|{LessRegex})\s*((el|las?|los)\s+)?(?<number2>({NumberSplitMark}.)+)
   references: [ LessOrEqual, LessRegex, NumberSplitMark ]
+OneNumberRangeLessRegex1LB: !nestedRegex
```

Comment:
What does LB stand for? Wouldn't it be possible to treat it as a prefix instead of altering all regexes?

Reply:
It stands for LookBehind. The problem is that the solution breaks the build in Java, so the new regex is only used in .NET